Lib/tempfile.py

```diff
@@ -477,10 +477,7 @@ def close(self):
                     self.cleanup()
 
     def __del__(self):
-        close_called = self.close_called
```

Comment:
This shouldn't be removed. We *want* to warn when there is a resource that is not closed. What should be fixed are the tests where we explicitly close the corresponding resources.